db/external_sst_file_test.cc

```diff
@@ -27,6 +27,7 @@ class ExternalSSTTestEnv : public EnvWrapper {
  public:
   ExternalSSTTestEnv(Env* t, bool fail_link)
       : EnvWrapper(t), fail_link_(fail_link) {}
```

Comment:
why we do not have static const char* kClassName() here?

Reply:
Added one.  It is not really needed since this Env is not referred to externally (the class is not registered with the ObjectRegistry or created by name elsewhere)

Reply:
Yeah, I realized that they are just locally used in the test. But just want to make all of them consistent.